index.bs

```diff
@@ -1409,6 +1958,44 @@ partial interface MLGraphBuilder {
     computed as the sum of all the input sizes of the same dimension.
 </div>
 
+<details open>
+  <summary>
+    The {{MLGraphBuilder/concat(inputs, axis)}} steps are:
+  </summary>
+  <div algorithm=concat class=algorithm-steps>
+    <div class="note">
+        The permissions and context validity have been checked by [[#api-mlgraphbuilder-constructor]] steps.
+    </div>
+    1. [=Assert=]: the type of |inputs| is sequence of {{MLOperand}} objects.
+    1. [=Assert=]: the type of |axis| is `unsigned long`.
+    1. [=Assert=]: the shape, i.e. {{MLOperandDescriptor/dimensions}}) of each operand in |inputs| is the same, except on the dimension given by |axis| on which they are concatenated.
+    1. [=Assert=]: the {{MLOperandDescriptor/type}} of each operand in |inputs| is the same.
+    1. If any of the following steps fail, then [=exception/throw=] a "{{DataError}}" {{DOMException}}.
+        1. If |axis| is greater than or equal to the <a>rank</a> of |inputs|, fail.
```

Comment:
`the <a>rank</a> of |inputs|[0].{{MLOperandDescriptor/dimensions}}` to replace `the <a>rank</a> of |inputs|`? The latter one means the number of inputs if I understand correctly.